commands/core/queue.drush.inc

```diff
@@ -46,16 +46,13 @@ function queue_drush_command() {
  * Validation callback for drush queue-run.
  */
 function drush_queue_run_validate($queue_name) {
-  $queue = drush_queue_get_class();
-  $queue->validate('queue-run');
-}
-
-/**
- * Validation callback for drush queue-list.
- */
-function drush_queue_list_validate() {
-  $queue = drush_queue_get_class();
-  $queue->validate('queue-list');
+  try {
+    $queue = drush_queue_get_class();
+    $queue->getInfo($queue_name);
+  }
+  catch (\Drush\Queue\QueueException $exception) {
```

Comment:
Is this really needed? In modern Drush, the command callback is wrapped in a try/catch so any exception should get caught there.